packages/yoastseo/spec/fullTextTests/testTexts/hu/hungarianPaper.html

```diff
@@ -49,7 +49,7 @@ <h1 id="firstHeading" class="firstHeading" lang="hu">Világegyetem</h1>
         <div id="jump-to-nav"></div>
         <a class="mw-jump-link" href="#mw-head">Ugrás a navigációhoz</a>
         <a class="mw-jump-link" href="#searchInput">Ugrás a kereséshez</a>
-        <div id="mw-content-text" lang="hu" dir="ltr" class="mw-content-ltr"><div class="mw-parser-output"><div class="thumb tright"><div class="thumbinner" style="width:302px;"><a href="/wiki/F%C3%A1jl:Hubble_ultra_deep_field_high_rez_edit1.jpg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/0/0d/Hubble_ultra_deep_field_high_rez_edit1.jpg/300px-Hubble_ultra_deep_field_high_rez_edit1.jpg" decoding="async" width="300" height="300" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/0/0d/Hubble_ultra_deep_field_high_rez_edit1.jpg/450px-Hubble_ultra_deep_field_high_rez_edit1.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/0/0d/Hubble_ultra_deep_field_high_rez_edit1.jpg/600px-Hubble_ultra_deep_field_high_rez_edit1.jpg 2x" data-file-width="6200" data-file-height="6200" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/F%C3%A1jl:Hubble_ultra_deep_field_high_rez_edit1.jpg" class="internal" title="Nagyítás"></a></div>A <a href="/wiki/Hubble_%C5%B1rt%C3%A1vcs%C5%91" title="Hubble űrtávcső">Hubble űrtávcső</a> felvételén látható legtávolabbi galaxisok fénye kb. 13 milliárd évvel ezelőtt indult el felénk</div></div></div>
+		<div id="mw-content-text" lang="hu" dir="ltr" class="mw-content-ltr"><div class="mw-parser-output"><div class="thumb tright"><div class="thumbinner" style="width:302px;"><a href="/wiki/F%C3%A1jl:Hubble_ultra_deep_field_high_rez_edit1.jpg" class="image"><img alt="" src="//upload.wikimedia.org/wikipedia/commons/thumb/0/0d/Hubble_ultra_deep_field_high_rez_edit1.jpg/300px-Hubble_ultra_deep_field_high_rez_edit1.jpg" decoding="async" width="300" height="300" class="thumbimage" srcset="//upload.wikimedia.org/wikipedia/commons/thumb/0/0d/Hubble_ultra_deep_field_high_rez_edit1.jpg/450px-Hubble_ultra_deep_field_high_rez_edit1.jpg 1.5x, //upload.wikimedia.org/wikipedia/commons/thumb/0/0d/Hubble_ultra_deep_field_high_rez_edit1.jpg/600px-Hubble_ultra_deep_field_high_rez_edit1.jpg 2x" data-file-width="6200" data-file-height="6200" /></a>  <div class="thumbcaption"><div class="magnify"><a href="/wiki/F%C3%A1jl:Hubble_ultra_deep_field_high_rez_edit1.jpg" class="internal" title="Nagyítás"></a></div>A <a href="/wiki/Hubble_%C5%B1rt%C3%A1vcs%C5%91" title="Hubble űrtávcső">Hubble űrtávcső</a> felvételén látható legtávolabbi galaxisok fénye kb. 13 milliárd évvel ezelőtt indult el felénk</div></div></div>
             <p>A <b>világegyetem</b> (latinosan <i>univerzum</i>) <a href="/wiki/Csillag%C3%A1szat" title="Csillagászat">csillagászati</a> fogalom, minden létező összességét jelenti.
```

Comment:
There's no difference between the two <div> elements. I removed the section at first, but then realized it's easier to adapt the expected results of some assessments instead, so I added it back.